### PR TITLE
[enhancement] #266 theme panel based on palette

### LINKED ISSUE #266: [Enhancement] theme panel based on palette.
Take inspiration from quartz code.

### PR DESCRIPTION
**Pull request by JavaLavaMT**
closes #266

### CODE REVIEW

**Comment by James1x0** [comment]
@JavaLavaMT Screenshot plz thnx

**Comment by JavaLavaMT** [comment]
screen shot 2018-08-03 at 11 58 20 am

File: app/controllers/application.js [hunk 1]
```diff
@@ -5,6 +5,10 @@ import { on } from '@ember/object/evented';
 import { inject as service } from '@ember/service';
 import $ from 'jquery';
 import ENV from 'granite/config/environment';
+import fadeRgb from 'granite/utils/fade-rgb';
+import { darken, lighten } from 'granite/utils/mul-rgb';
+import { htmlSafe } from '@ember/string';
+
```
Comment (James1x0) [inline app/controllers/application.js]: ✂️

File: app/controllers/application.js [hunk 2]
```diff
@@ -64,6 +69,16 @@ export default Controller.extend({
     }
   })),
 
+
```
Comment (James1x0) [inline app/controllers/application.js]: ✂️

File: app/templates/application.hbs [hunk 3]
```diff
@@ -1,6 +1,7 @@
 {{notification-container notifications=notifications position="bottom"}}
 
-<div class="ui main">
+<div class="ui main" style="background: {{backdrop}} ">
+  {{log backdrop.palette}}
```
Comment (James1x0) [inline app/templates/application.hbs]: Hep

**Pull request merged**